lib/dpl/providers/npm.rb

```diff
@@ -42,7 +45,7 @@ def login
       end
 
       def deploy
-        shell :deploy
+        shell run_script? ? :run : :publish
```

Comment:
I am not a big fan of this construct. It caught me by surprise how difficult it is to reason. Can we do a more straightforward

```ruby
if run_script?
  shell :run
else
  shell :publish
end
```

Reply:
sure thing!